src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringCalculatorTest {
+
+    @Test
+    @DisplayName("빈 문자열은 0으로 반환한다.")
+    void emptyStringIsZeroTest() {
+        assertThat(StringCalculator.isEmpty("")).isTrue();
+        assertThat(StringCalculator.addOperation("")).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(StringCalculator.addOperation("")).isZero();
```
로 써볼 수도 있을 것 같습니다 ㅎㅎ

Reply:
헐 코드 까지 작성해주시다니 감사드립니다 ㅠㅜ